metashare/api/jobs.py

```diff
@@ -55,6 +55,19 @@ def report_errors_on_provision(scratch_org):
         raise
 
 
+@contextlib.contextmanager
+def report_errors_on_check_changes(scratch_org):
+    try:
+        yield
+    except Exception as e:
+        async_to_sync(report_scratch_org_error)(
```

Comment:
I've noticed that the errors from SF are in a particular format:

```
[{'message': 'The signup request failed because this organization has reached its active scratch org limit', 'errorCode': 'LIMIT_EXCEEDED', 'fields': []}]
```

Would it make sense for the WS event to include just `e[0].message` or whatever instead of the entire thing?

Reply:
@davisagli can we reliably expect to get just one error, and extract it this way? Or do we need to account for the possibility of multiple errors in the list.

Reply:
Given that https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/errorcodes.htm says "The response body _usually_ contains...", I would avoid assuming too much.

Reply:
Okay... can we do this only if there's one error, and it has a `message` property?

Reply:
@jgerigmeyer I feel like that would be more annoying on both the front and back end, to have to special case the one-item-array case? It would be moderately straightforward on the backend, but then the frontend would have to detect whether it got an array or an object and change display logic accordingly, right?

Reply:
No, the front end is just getting a string as it is now. I just think if the majority of the time we're getting a one-item array with a `message` property, let's default to displaying that to users.

Doing that check on the back end seems simplest to me, but if you prefer sending an actual array instead of a string I could do the check on the front end. 🤷‍♂

Reply:
OK; I thought it'd be more annoying for you, and wanted to make sure it wasn't. I'm happy to do it on the backend.